config/config.go

```diff
@@ -147,6 +148,13 @@ type ProxyProtocol struct {
 	HeaderTimeout int `toml:"header-timeout" json:"header-timeout"`
 }
 
+// TiKVClient is the config for tikv client.
+type TiKVClient struct {
+	// GrpcConnectionCount is the max gRPC connections that will be established
+	// with each tikv-server.
+	GrpcConnectionCount int `toml:"grpc-connection-count" json:"grpc-connection-count"`
+}
+
 var defaultConf = Config{
```

Comment:
Why not add it to defaultConf?

Reply:
defaultConf does not use tikv store.

Reply:
You can see the defalt config has a lot of `Enabled: false`, but still has other default values set.